src/main/java/core/basesyntax/JavaDateTimeApi.java

```diff
@@ -74,16 +105,21 @@ public LocalDate addWeeks(LocalDate localDate, Integer numberOfWeeks) {
      * - "someDate is today" - если someDate - сегодня
      */
     public String beforeOrAfter(LocalDate someDate) {
-        return someDate + "is today";
+        LocalDate nowDate = LocalDate.now();
+        return nowDate.isAfter(someDate) ? someDate + " is before " + nowDate
```

Comment:
let's use relevant methods for example someDate.isAfter(nowDate) ? someDate + " is after " + nowDate